integration/polygon-pos/eth_getLogs/readme.md

```diff
@@ -0,0 +1,2 @@
+test_01.json - 
+"bor logs" topis and log addressed are reported back -- indicates if logaddr/logtopics are indexed properly on system txs.
```

Comment:
We usually add the test description if any within the response file itself, e.g. https://github.com/erigontech/rpc-tests/blob/main/integration/mainnet/eth_call/test_28.json#L5